frontend/src/components/pagination.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  ChevronLeft,
+  ChevronRight,
+  ChevronsLeft,
+  ChevronsRight
+} from "lucide-react";
+import { Button } from "./ui/button";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue
+} from "./ui/select";
+
+export type PaginationProps = {
+  totalPages: number;
+  currentPage: number;
+  perPage: number;
+  className?: string;
+  onChangePage: (page: number) => void;
+  onChangePerPage: (perPage: number) => void;
+};
+
+export function Pagination({
+  totalPages,
+  currentPage,
+  perPage,
+  className = "",
+  onChangePage,
+  onChangePerPage
+}: PaginationProps) {
+  return (
+    <div className={`flex items-center justify-end px-2 ${className}`}>
```

Comment:
```suggestion
    <div className={cn("flex items-center justify-end px-2", className)}>
```

use `cn` it handles `undefined | false | null` values for automatically